.goreleaser.yml

```diff
@@ -54,6 +54,22 @@ blobs:
     folder: "dagger/releases/{{ .Version }}"
 
 publishers:
+  - name: publish-install-sh
+    cmd: sh -c "aws s3 cp install.sh s3://{{ .Env.AWS_BUCKET }}/dagger/install.sh"
```

Comment:
From what I can tell, there's currently no automation for this right now? This is what prevented `install.ps1` from being updated in the docs in the first place.

If so, I think this is the right way to do it, we should have goreleaser release the install script.

Reply:
makes sense to me if it works and has thumbs up from @matipan or @gerhard 👍

Reply:
We do this manually because we need to invalidate CloudFront too. FTR:
- https://github.com/dagger/dagger/pull/5829

If we want to do this properly, we should add it to a function as a follow-up to https://github.com/dagger/dagger/pull/6843

If we want to do this quick & dirty, we need to also invalidate CloudFront. Note-worthy past comments:
- https://github.com/dagger/dagger/pull/1819#issuecomment-1075508169
- https://github.com/dagger/dagger/pull/1819#issuecomment-1077651647
- https://github.com/dagger/dagger/pull/1819#issuecomment-1078511779

Reply:
Sounds good! Hadn't realized that, I've updated the RELEASING guide to include info about doing this.